src/api/authorize/use-p2p-country-list.tsx

```diff
@@ -0,0 +1,25 @@
+import { useAuthorizeQuery } from '../../base';
+import { TSocketQueryOptions } from '../../base/use-query';
+
+/**
+ * A custom hook that returns an object containing the list of countries available for P2P trading.
+ *
+ * For returning details of a specific country, the country code can be passed in the payload.
+ * @example
+ *  useP2PCountryList({
+ *      payload: {
+ *          country: 'id'
+ *      }
+ *  })
+ */
```

Comment:
Lets remove this documentation till @yashim-deriv comes up with a plan for it. Since all the hooks are automated by scripts, it might mess up the automation process

Reply:
ahh got it, makes sense it might mess up the script 😅